tests/EntryTest.php

```diff
@@ -12,12 +12,36 @@ class EntryTest extends \PHPUnit_Framework_TestCase
 
     public function testEntryTTL()
     {
-        $myaddr = new Address('mybin', 'mykey');
-        $entry = new Entry(0, 'mypool', $myaddr, 'value', $_SERVER['REQUEST_TIME'], $_SERVER['REQUEST_TIME'] + 1);
-        $this->assertEquals(1, $entry->getTTL());
+        $this->assertEquals(1, $this->getEntry(1)->getTTL());
 
         // TTL should be zero for already-expired items.
-        $old_entry = new Entry(0, 'mypool', $myaddr, 'value', $_SERVER['REQUEST_TIME'], $_SERVER['REQUEST_TIME'] - 1);
-        $this->assertEquals(0, $old_entry->getTTL());
+        $this->assertEquals(0, $this->getEntry(-1)->getTTL());
+
+        // TODO: How to classify this type of item?
+        $this->assertEquals(0, $this->getEntry(0)->getTTL());
+    }
+
+    public function testExpiry()
+    {
+        $this->assertTrue($this->getEntry(-1)->isExpired());
+        $this->assertFalse($this->getEntry(1)->isExpired());
+
+        // TODO: How to classify this one?
+        $this->assertFalse($this->getEntry(0)->isExpired());
```

Comment:
Is expired and getTTL are a bit strange for relative times. They are the same for older and newer times, bu the difference comes from the current moment in time. The function getTTL says it has no time to live (but it is not dead yet) and the isExpired follows the same idea at the moment.